dotnet/src/SemanticKernel.Core/Functions/SKFunctionFromPrompt.cs

```diff
@@ -134,18 +134,18 @@ internal sealed class KernelFunctionFromPrompt : KernelFunction
     /// <inheritdoc/>
     protected override async Task<FunctionResult> InvokeCoreAsync(
         Kernel kernel,
-        SKContext context,
+        ContextVariables variables,
         AIRequestSettings? requestSettings = null,
         CancellationToken cancellationToken = default)
     {
-        this.AddDefaultValues(context.Variables);
+        this.AddDefaultValues(variables);
```

Comment:
Many `new ContextVariable()` code typing will be prevented with this change.

```suggestion
        ContextVariables variables = null,
        AIRequestSettings? requestSettings = null,
        CancellationToken cancellationToken = default)
    {
        variables ??= new ContextVariables();
        this.AddDefaultValues(variables);
```

Reply:
I'll do it in one of the PRs when replacing parameters of ContextVariables type by KernelFunctionArguments or the one afterwards.